keypatch.py

```diff
@@ -16,12 +16,16 @@
 # To revert (undo) the last patching, choose menu "Edit | Keypatch | Undo last patching".
 # To check for update version, choose menu "Edit | Keypatch | Check for update".
 
-import idc
-import idaapi
+import os
 import re
 import json
 from keystone import *
-
+import idc
+import idaapi
+try:
+    from idc import GetOpType, GetOpnd, ItemEnd
+except ImportError:
+    from idc_bc695 import GetOpType, GetOpnd, ItemEnd
```

Comment:
where is this idc_bd695 from?

Reply:
As you are aware, there are major API changes between 32-bit IDA 6.x and 64-bit IDA 7.0.  As a way to aid plugin porting, Hex-Rays has provided a backwards compatibility layer for some of the old APIs in 64-bit IDA 7.0.  The ```idc_bc695``` module provides these wrappers.

Reply:
Certainly the code which calls these functions could be refactored and wrapped in conditional statements, but this seemed to be the cleaner approach.